src/Resources/doc/clients.md

```diff
@@ -49,6 +49,20 @@ csa_guzzle:
             # ...
 ```
 
+You can also ```extend``` another client.
+
+```yml
+csa_guzzle:
+    clients:
+        parent_one:
+            config: 
+                 headers: 
+                     Accept: application/json
+        my_client:
+            extends: parent_one
+            # ...
+```
+
```

Comment:
I meant give an example showing how to do the same thing using a service definition in XML / YAML.

E.g.

```yml
services:
    my_service:
        class: GuzzleHttp\Client
        tags:
            - { name: 'csa_guzzle.client' }
            - { name: 'csa_guzzle.inheritance', parent: 'my_parent_service' }        
```